consistency-checker/src/main/java/com/redhat/lightblue/migrator/consistency/JobStatus.java

```diff
@@ -0,0 +1,74 @@
+package com.redhat.lightblue.migrator.consistency;
+
+/**
+ *
+ * JobStatus can be one of the following values:
+ *      NEW                - Means the Job just created and it isn't ready to be executed
+ *      READY              - Means the Job is ready to be executed but not in execution
```

Comment:
Status is on a job execution record.  This is only created when a job is being executed.  Status of NEW and READY will not be part of execution.  NEW is implied by the migrationJob existing in the db without any elements in jobExecutions.  READY is implied by the migrationJob existing in the db without any elements in jobExecutions and whenAvailableDate <= new Date()